### ISSUE TITLE
Support for oneOf in openapi generator

### ISSUE THREAD

**Issue by Tetraquark** [icerockdev/moko-network#83]
Need to implement supporting `oneOf` constructions like this:

```
User:
  type: object
  properties:
    birth_date:
      oneOf:
      - $ref: "#/components/schemas/UserBirthday"
      - description: Дата рождения
          type: string
          format: date
```

**Comment by Alex009** [comment]
Go try to implement it as `JsonElement` in output:
```
@Serializable
data class User(
    @SerialName("birth_date")
    val birthDate: JsonElement
)
```

and in consumer app this field will be read in runtime

**Comment by Alex009** [comment]
will be available in 0.11.0
